specification/columns/skumeter.md

```diff
@@ -0,0 +1,43 @@
+# SKU Meter
+
+The SKU Meter describes the functionality being metered or measured by a particular SKU in a charge.
+
+Providers often have billing models in which multiple SKUs exist for a given service to describe and bill for different functionalities for that service. For example, an object storage service may have separate SKUs for functionalities such as object storage, API requests, data transfer, encryption, object management. This field helps practitioners understand which functionalities are being metered by the different SKUs that appear in a FOCUS dataset.
+
+The SkuMeter column adheres to the following requirements:
+
+* SkuMeter MUST be present in the billing data when when the provider includes a [SkuId](#skuid).
+* SkuMeter MUST be of type String.
+* SkuMeter MUST be null when SkuId is null.
+* SkuMeter SHOULD NOT be null when SkuId is not null.
+* SkuMeterName SHOULD remain consistent over time for a given SkuId.
```

Comment:
References "SkuMeterName" vs "SkuMeter" though I know that the naming is up for debate currently

Reply:
thanks - fixed